app/_layout.tsx

```diff
@@ -111,26 +111,27 @@ function RootLayoutNav() {
   const colorScheme = useColorScheme()
 
   /** Subscribe to SSE from Pocketbase */
-  useEffect(() => {
-    // Sanity check. Records can be written in non-real-time
-    // const result = await pocketbase.collection('test').getFullList()
-
-    // ! no async
-    pocketbase.collection('test').subscribe('*', (e) => {
-      console.log('Realtime event received:', e.action)
-      console.log('Record data:', e.record)
-    })
-
-    return () => {
-      console.log('unsubscribe')
-      pocketbase.collection('test').unsubscribe('*')
-    }
-  }, [])
+  // useEffect(() => {
+  //   // Sanity check. Records can be written in non-real-time
+  //   // const result = await pocketbase.collection('test').getFullList()
+
+  //   // ! no async
+  //   pocketbase.collection('test').subscribe('*', (e) => {
+  //     console.log('Realtime event received:', e.action)
+  //     console.log('Record data:', e.record)
+  //   })
+
+  //   return () => {
+  //     console.log('unsubscribe test')
+  //     pocketbase.collection('test').unsubscribe('*')
+  //   }
+  // }, [])
```

Comment:
This commented block appears to be unused - we could clean this up by removing it.